packages/spacecat-shared-data-access/test/it/db.test.js

```diff
@@ -134,22 +138,28 @@ describe('DynamoDB Integration Test', async () => {
     process.env.AWS_ACCESS_KEY_ID = 'dummy';
     process.env.AWS_SECRET_ACCESS_KEY = 'dummy';
 
-    dynamoDbLocalProcess = DynamoDbLocal.spawn({
+    dynamoDbLocalProcess = spawn({
+      detached: true,
+      stdio: 'inherit',
       port: 8000,
       sharedDb: true,
     });
 
-    await sleep(10000); // give db time to start up
-
-    await generateSampleData(
-      TEST_DA_CONFIG,
-      NUMBER_OF_ORGANIZATIONS,
-      NUMBER_OF_SITES,
-      NUMBER_OF_SITES_CANDIDATES,
-      NUMBER_OF_AUDITS_PER_TYPE_AND_SITE,
-      NUMBER_OF_TOP_PAGES_FOR_SITE,
-      NUMBER_OF_KEY_EVENTS_PER_SITE,
-    );
+    await sleep(5000); // give db time to start up
```

Comment:
I would rather keep this as 10s. We had many cases where 5s just didn't cut it

Reply:
i want to see whether the major update speeds things up in the interest of spent action units. if it turns out to be inappropriate, we set back to 10